dev-docs/FrequentTasks.md

```diff
@@ -208,3 +208,7 @@ chef-server/src/bookshelf/rebar.lock
 chef-server/src/oc_erchef/rebar.lock
 chef-server/src/oc_bifrost/rebar.lock
 ```
+
+## Buildkite Artifacts (omnibus/adhoc)
+
+If you want an artifact to appear in Artifactory's unstable channel, you must now set `PUBLISH_TO_ARTIFACTORY=true` in the Environment Variables section of the Buildkite web GUI.  Select New Build, then select Options in the popup to open the Environment Variables section.
```

Comment:
Is there a way this can be done for the nightlies? It would be good to make a note of that.

Reply:
I'll try to find that out.  I'm guessing it would require a change to whatever buildkite script is running this nightly.

Reply:
Just a thought, but I wonder if you could just add it as an environment variable to your `.expeditor/config.yml`:


```
diff --git a/.expeditor/config.yml b/.expeditor/config.yml
index 36962f703..ac772214e 100644
--- a/.expeditor/config.yml
+++ b/.expeditor/config.yml
@@ -43,6 +43,7 @@ pipelines:
  - omnibus/adhoc:
       definition: .expeditor/release.omnibus.yml
       env:
         - ADHOC: true
+        - PUBLISH_TO_ARTIFACTORY: true
   - post-promote:
       description: "Generate and upload release manifest"
       definition: .expeditor/post-promote.pipeline.yml

```